docs/CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Fix
+---
+* Make not always present ``cuffnorm`` outputs not required and fix
+  what was done in the previous ``cuffnorm`` commit
```

Comment:
Maybe rephrase this to "...and revert previous ``cuffnorm`` commit (commit hash)"